tests/envs/test_base_env.py

```diff
@@ -4,34 +4,13 @@
 
 from predicators.src import utils
 from predicators.src.envs import BaseEnv, create_new_env, get_or_create_env
+from predicators.tests.approaches.test_oracle_approach import ENV_NAME_AND_CLS
 
 
 def test_env_creation():
     """Tests for create_new_env() and get_or_create_env()."""
     utils.reset_config({"num_train_tasks": 5, "num_test_tasks": 5})
-    for name in [
-            "cover",
-            "cover_typed_options",
-            "cover_hierarchical_types",
-            "cover_regrasp",
-            "cluttered_table",
-            "blocks",
-            "playroom",
-            "painting",
-            "tools",
-            "repeated_nextto",
-            "repeated_nextto_single_option",
-            "repeated_nextto_painting",
-            "cover_multistep_options",
-            "cover_multistep_options_fixed_tasks",
-            "cover_multistep_options_holding",
-            "pybullet_cover",
-            "pybullet_blocks",
-            "pddl_blocks_fixed_tasks",
-            "pddl_blocks_procedural_tasks",
-            "touch_point",
-            "stick_button",
-    ]:
+    for name, _ in ENV_NAME_AND_CLS:
```

Comment:
is there a reason to not make this one parameterized as well?

Reply:
too lazy to pull the last few lines (`with pytest.raises(NotImplementedError): create_new_env("Not a real env")`) into a new function